### PR TITLE
.travis: disable go modules when installing dependencies

### PR DESCRIPTION
**Pull request by aanm**
Fixes: d1fecef5d46f ("vendor: add go-mod support")
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9462)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -323,7 +323,7 @@ gofmt:
 
 govet:
 	@$(ECHO_CHECK) vetting all GOFILES...
-	$(QUIET)$(GO) vet \
+	$(QUIET) $(GOLIST) $(GO) vet \
```
Comment (ianvernon) [inline Makefile:326]: This change appears unrelated?

Comment (aanm) [reply Makefile:326]: It's related.

Comment (ianvernon) [reply Makefile:326]: Can you please specify why? I'm not doubting you, but context would be helpful here.

Comment (aanm) [reply Makefile:326]: My fault, I've updated the commit description.

**Review by ianvernon — APPROVED** [review]

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please
